openstates/co/utils/email.py

```diff
@@ -0,0 +1,4 @@
+def mailto_to_email(mailto_url):
+    """Return a single email extracted from 'mailto:name@domain.com'"""
+    email = mailto_url.lstrip('mailto:')
```

Comment:
lstrip will be overzealous here since it interprets its argument as a sequence of chars to strip

In [1]: 'mailto:tom@gmail.com'.lstrip('mailto:')
Out[1]: '@gmail.com'

probably better to check for mailto: prefix and .replace('mailto:', '')